test/fixture-auth-provider-cmd/app1/creds.config

```diff
@@ -0,0 +1,3 @@
+fake_id=app1-ASIA6ODU3VLNW2YGRZ12
```

Comment:
We can leave these as a lot less real so that it doesn't scare the secrets scanner 😄 

It can be `fake_id=fake_id`, etc.

Reply:
Fixed, thanks)